delivery_roulier_geodis_fr/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# from . import test_geodis_labels
```

Comment:
This file must be deleted. I don't really know which commit makes this change, but it should be deleted in that commit.

Reply:
Yes, that commit should not be in this PR because it does not belong to this module.